.circleci/config.yml

```diff
@@ -1,50 +1,79 @@
 ---
-version: 2
+version: 2.1
+
+executors:
+  node:
+    docker: [{image: 'circleci/node:10'}]
+
+  ruby:
+    docker: [{image: 'circleci/ruby:2.6.1'}]
+
+commands:
+  danger:
+    description: 'Run Danger'
+    steps:
+      - run:
+          command: if ! [[ $CIRCLE_PR_NUMBER ]]; then yarn run danger --id $task; fi
+          when: always
+          environment:
+            DANGER_DISABLE_TRANSPILATION: 'true'
+
+  embed-env-vars:
```

Comment:
this one could just vanish, tbh

Reply:
@hawkrives are you saying we do not need this anymore?

Reply:
I agree with @hawkrives. We're no longer relying on the `.env.js` file since api keys have been moved to `ccc-server`.

Reply:
Removed.